api/src/pcapi/routes/backoffice/templates/custom_reimbursement_rules/list/stats.html

```diff
@@ -0,0 +1,24 @@
+<turbo-frame data-turbo="false" id="reimbursement_stats">
+<div class="card h-100 shadow">
+  <div class="card-body row">
+    <div class="col-4">
+      <h5 class="fs-2 card-title">{{ stats.by_offerer or 0 }}</h5>
+      <p class="card-text fw-light small mb-0">
+        Tarifs dérogatoires sur des <b>structures</b>
```

Comment:
Pour pinailler, la phrase commence au chiffre :slightly_smiling_face: 
```suggestion
        tarifs dérogatoires sur des <b>structures</b>
```